pkg/types/validation/installconfig.go

```diff
@@ -38,7 +38,7 @@ func ValidateInstallConfig(c *types.InstallConfig, openStackValidValuesFetcher o
 	switch v := c.APIVersion; v {
 	case types.InstallConfigVersion:
 		// Current version
-	case "v1beta1", "v1beta2":
+	case "v1beta2":
```

Comment:
Do we need `v1beta2` any more? CI and hive are both using `v1beta3`.

Reply:
But I am fine if we want to merge this and do another PR while we confirm if `v1beta2` is needed.

Reply:
> Do we need `v1beta2` any more? CI and hive are both using `v1beta3`.

I'm not in a hurry to land this.  Can you link to their bumps for the new commit message?